CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## v0.16.0
+
+### Optimization
+
++ Logger Roller supports the custom Roller [@wenxuwan](https://github.com/wenxuwan)
++ StreamFilter adds new api SendHijackReplyWithBody [@wenxuwan](https://github.com/wenxuwan)
++ The configuration adds option of turning off the smooth upgrade. If the smooth ugrade is turned off, different MOSN processes can exist on a host at the same time [@cch123](https://github.com/cch123)
++ Optimize the MOSN integration test framework and add more unit tests. [@nejisama](https://github.com/nejisama) [@wangfakang](https://github.com/wangfakang) [@taoyuanyuan](https://github.com/taoyuanyuan)
```

Comment:
+ Logger Roller supports the custom Roller. [@wenxuwan](https://github.com/wenxuwan)
+ Add a SendHijackReplyWithBody API for streamFilter. [@wenxuwan](https://github.com/wenxuwan)
+ The configuration adds option of turning off the smooth upgrade. If the smooth upgrade is turned off, different instances of MOSN can be started on the same machine. [@cch123](https://github.com/cch123)
+ Optimize the MOSN integration test framework and add more unit test cases. [@nejisama](https://github.com/nejisama) [@wangfakang](https://github.com/wangfakang) [@taoyuanyuan](https://github.com/taoyuanyuan)
+ DirectResponse route configuration supports the update mode of XDS. [@wangfakang](https://github.com/wangfakang) 
+ Add a new field of TLSContext for clusterManager configuration. [@nejisama](https://github.com/nejisama)